### ISSUE TITLE
Error in postFlush if an auditable entity has a "resource" field

### ISSUE THREAD

**Issue by scapital** [oroinc/platform#818]
I see [this exception](https://github.com/oroinc/platform/blob/master/src/Oro/Component/MessageQueue/Client/MessageProducer.php#L80) if an entity has a field with "resource" type (entity was saved successfully to DB).

I think adding checking to [this code](https://github.com/oroinc/platform/blob/master/src/Oro/Bundle/DataAuditBundle/Service/EntityToEntityChangeArrayConverter.php#L96) will help and won't break anything.
I mean something like that
```
if (!is_scalar($sanitized)) {
    $sanitized = null;
}

return $sanitized;
```

Thanks

**Comment by mbessolov** [comment]
Thank you for the suggestion. We'll check if it can be done this way and will include in one of the next releases.

**Comment by anyt** [comment]
The fix merged to master, already released at `3.0.0`
https://github.com/oroinc/platform/commit/457e16600fec12989c19a60be43e92255d821a59

Thank you for contribution!
